src/Gang/AllGangs.ts

```diff
@@ -70,6 +39,12 @@ export function resetGangs(): void {
   };
 }
 
+export let AllGangs: Record<string, GangTerritory> = getDefaultAllGangs();
```

Comment:
Can this be const? (I know it was pre-existing)

Reply:
No, it can't, I am a dumb

Reply:
Technically it could be with some extra work. The mutation is not used outside of this package. The only mutation happens below from the reviver.